src/Polly.Core/Polly.Core.csproj

```diff
@@ -25,6 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Condition="!$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'netcoreapp3.1'))" />
+    <PackageReference Include="Microsoft.Bcl.TimeProvider" />
```

Comment:
Prior to RC.1 this was conditional, but it seemed to break locally after updating unless this was present for all TFMs.

Reply:
This is odd, we shouldn't be forced to add this for .NET 8

Reply:
This was fixed by adding .NET 8 target to `Polly.Testing`. 

`Microsoft.Bcl.TimeProvider` is still not included on .NET 8